### ISSUE TITLE
[meter] `aria-valuenow` should not be converted to percentage

### ISSUE THREAD

**Issue by brianle1301** [mui/base-ui#2263]
# Bug report

## Current behavior

`Meter` converts user provided `value` to a percentage within `max` and `min` for `aria-valuenow`, as can be seen on the documentation site:

<img width="618" height="106" alt="Image" src="https://github.com/user-attachments/assets/a342d9d0-c5ce-46d1-a6da-b1b813487373" />

This produces a value that is incorrect within the user provided range.

## Expected behavior

The user provided value should be used as is for `aria-valuenow`

## Reproducible example

https://base-ui.com/react/components/meter

## Base UI version

1.0.0-beta.1

## Which browser are you using?

N/A

## Which OS are you using?

N/A

## Which assistive tech are you using (if applicable)?

N/A

## Additional context

This is not a problem on the `Progress` component.
